### PR TITLE
Hide launcher interface scaling spin box

### PR DESCRIPTION
**Pull request by mikeiit**
Hide launcher interface scaling spin box when automatic scaling is on. This is related to #4805.

### CODE REVIEW

**Comment by mikeiit** [comment]
Is there a problem with the Mac build ci build?

**Comment by kambala-decapitator** [comment]
that's a "traditional" macOS failure not related to a project. Simply restart failed builds when you encounter it, just did it.

File: launcher/settingsView/csettingsview_moc.cpp [hunk 1]
```diff
@@ -838,9 +838,17 @@ void CSettingsView::on_sliderScalingCursor_valueChanged(int value)
...
-	ui->spinBoxInterfaceScaling->setValue(100);
-
+	if (checked)
+	{
+		ui->spinBoxInterfaceScaling->hide();
+	}
+	else
+	{
+		ui->spinBoxInterfaceScaling->show();
+		ui->spinBoxInterfaceScaling->setDisabled(false);
```
Comment (kambala-decapitator) [inline launcher/settingsView/csettingsview_moc.cpp:848]: if it's the only place where enabled/disabled state is changed, then this line isn't needed

Comment (mikeiit) [reply launcher/settingsView/csettingsview_moc.cpp:848]: Good point. I'm on vacation now. I can update it afterwards.

Comment (kambala-decapitator) [reply launcher/settingsView/csettingsview_moc.cpp:848]: FYI you can edit the file directly on GitHub in your fork/branch

Comment (IvanSavenko) [reply launcher/settingsView/csettingsview_moc.cpp:848]:
```suggestion
```

**Pull request merged**